wails-frontend/src/pages/backupKeyPairs.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react';
+import { promptRequest } from '../events/events';
+import { useLocation } from 'react-router-dom';
+import { handleCancel } from '../utils/utils';
+import { Password, Button } from '@massalabs/react-ui-kit';
+import { FiCopy } from 'react-icons/fi';
+
+function BackupKeyPairs() {
+  const [inputPassword, setInputPassword] = useState<string | undefined>(
+    undefined,
+  );
+  const [show, setShow] = useState<boolean>(false);
+  const [privateKey, setPkey] = useState<string | undefined>(undefined);
+
+  let { state } = useLocation();
+  const req: promptRequest = state.req;
+
+  const baselineStr = 'Enter your password to show your private key';
+
+  function handleCopy() {
+    console.log('copy');
+  }
+
+  function handleShow() {
+    if (!inputPassword || !inputPassword.length) {
+      return;
+    }
+
+    if (!privateKey) {
+      // TODO: validate private key
+      setPkey(inputPassword);
+      setShow(!show);
+      setInputPassword('');
+      return;
+    }
+  }
+
+  function updateInput(e: React.ChangeEvent<HTMLInputElement>) {
+    setInputPassword(e.target.value);
+  }
```

Comment:
I don't understand the reason of that TBH... 🤔

Reply:
Done